### PR TITLE
limit DAG file size & optimize logs

### PR DESCRIPTION
**Pull request by xiaoch05**

### CODE REVIEW

File: pkg/ethashproof/cache.go [hunk 1]
```diff
@@ -37,28 +43,46 @@ func (self *DatasetMerkleTreeCache) Print() {
 
 func getHomeDir() string {
 	usr, err := user.Current()
-	if err != nil {
-		log.Fatal(err)
-	}
+    log.Assert(err)
```
Comment (HackFisher) [inline pkg/ethashproof/cache.go:46]: format

File: pkg/ethashproof/dag_merkle_root.go [hunk 2]
```diff
@@ -48,10 +49,10 @@ func processDuringRead(f *os.File, startIn128Res int, fullSizeIn128Res uint32, m
 // 4. Return merkle root
 func CalculateDatasetMerkleRoot(epoch uint64, saveCache bool) (mtree.Hash, error) {
 	blockno := epoch * 30000
-	fmt.Printf("Make the dag\n")
+    log.Info("Make the dag epoch %v", epoch)
```
Comment (HackFisher) [inline pkg/ethashproof/dag_merkle_root.go:52]: format

**Review by HackFisher — CHANGES_REQUESTED** [review]

**Comment by HackFisher** [comment]
@xiaoch05 I request some changes regarding format.

@wuminzhe Please help review this pull request.

File: pkg/ethashproof/cache.go (hunk 1 above)
Comment (xiaoch05) [reply pkg/ethashproof/cache.go:46]: ok, fix it.

**Review by wuminzhe — APPROVED** [review]

**Review by HackFisher — APPROVED** [review]

**Pull request merged**